agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/instrumentation/sdk/BytecodeUtilImpl.java

```diff
@@ -55,29 +55,30 @@
 // supporting all properties of event, metric, remove dependency and page view telemetry
 public class BytecodeUtilImpl implements BytecodeUtilDelegate {
 
-    private static final Tracer tracer = OpenTelemetry.getGlobalTracer("");
-
     private static final Logger logger = LoggerFactory.getLogger(BytecodeUtilImpl.class);
 
     private static final AtomicBoolean alreadyLoggedError = new AtomicBoolean();
 
     @Override
-    public void trackEvent(String name, Map<String, String> properties, Map<String, Double> metrics) {
+    public void trackEvent(String name, Map<String, String> properties, Map<String, String> tags, Map<String, Double> metrics) {
 
         if (Strings.isNullOrEmpty(name)) {
             return;
         }
         EventTelemetry telemetry = new EventTelemetry(name);
-        MapUtil.copy(properties, telemetry.getContext().getProperties());
-        MapUtil.copy(metrics, telemetry.getMetrics());
+        telemetry.getProperties().putAll(properties);
```

Comment:
can you extract these 5 lines out as a private method to be used everywhere else?

Reply:
which 5 lines? I think there are only 2 lines that can be extracted:

```
        telemetry.getProperties().putAll(properties);
        telemetry.getContext().getTags().putAll(tags);
```

and not sure if worth it for these 2 lines

Reply:
the same comment is repeated in multiple places.. since you deleted it.. that's fine.

Reply:
oh, right, thx 👍